cms/djangoapps/contentstore/tests/tests.py

```diff
@@ -19,87 +19,20 @@
 from xmodule.modulestore.tests.factories import CourseFactory
 
 
```

Comment:
**Note to Reviewers:** The tests in this file were simplified to use the `users` created in `ModuleStoreTestCase` instead of needing to create and activate new ones.  The focus of these tests are _not_ to end-to-end test account creation/activation, but other functionality.